examples/appliance_snmpv3_user/import.tf

```diff
@@ -0,0 +1,12 @@
+provider "oneview" {
```

Comment:
Is this import required? As we create and update the same resource, main.tf and update.tf are sufficient.

Reply:
This is for user and not part of automation